linera-storage-service/src/common.rs

```diff
@@ -16,18 +18,39 @@ const TEST_SHARED_STORE_MAX_CONCURRENT_QUERIES: usize = 10;
 const TEST_SHARED_STORE_MAX_STREAM_QUERIES: usize = 10;
 
 #[derive(Debug, Error)]
-pub enum SharedContextError {
+pub enum ServiceContextError {
     /// Not matching entry
     #[error("Not matching entry")]
     NotMatchingEntry,
 
+    /// Failed to find the storage_service_server binary
+    #[error("Failed to find the storage_service_server binary")]
+    FailedToFindStorageServiceServerBinary,
+
     /// gRPC error
     #[error(transparent)]
     GrpcError(#[from] Status),
 
     /// Transport error
     #[error(transparent)]
     TransportError(#[from] tonic::transport::Error),
+
+    /// An error occurred while doing BCS serialization.
+    #[error("failed to serialize value to calculate its hash")]
```

Comment:
Is hashing really the only case where serialization can fail?

Reply:
Rewritten, sorry about that.